### PR TITLE
Add new KV feature gate: kvWithHostPassthroughCPU

### PR DESCRIPTION
**Pull request by nunnatsa**
Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

**Release note**:

```release-note
Add new KV feature gate: kvWithHostPassthroughCPU
```

### CODE REVIEW

File: pkg/controller/operands/kubevirt.go [hunk 1]
```diff
@@ -24,15 +24,17 @@ const (
 )
 
 const (
-	kvHotplugVolumes = "HotplugVolumes"
-	cmFeatureGates   = "DataVolumes,SRIOV,LiveMigration,CPUManager,CPUNodeDiscovery,Sidecar,Snapshot"
+	kvHotplugVolumes         = "HotplugVolumes"
+	kvWithHostPassthroughCPU = "WithHostPassthroughCPU"
```
Comment (enp0s3) [inline pkg/controller/operands/kubevirt.go:28]: Can we add `kvWithHostModelCPU = "WithHostModelCPU"? We need to manage them both.

File: pkg/controller/operands/kubevirt.go [hunk 2]
```diff
@@ -24,15 +24,17 @@ const (
...
+	cmFeatureGates           = "DataVolumes,SRIOV,LiveMigration,CPUManager,CPUNodeDiscovery,Sidecar,Snapshot"
 )
 
 var (
 	// managedKvFeatureGates - list of KV feature gates that can be set/clear by adding/remove them
 	// from HyperConverged CR
 	managedKvFeatureGates = []string{
 		kvHotplugVolumes,
+		kvWithHostPassthroughCPU,
```
Comment (enp0s3) [inline pkg/controller/operands/kubevirt.go:37]: Following my previous comment, can we add `kvWithHostModelCPU` to the list?

**Review by enp0s3 — CHANGES_REQUESTED** [review]

**Comment by nunnatsa** [comment]
Closed in favor of #1063

**Pull request closed without merging**